_posts/2024-04-21-Prosper_Contibution.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+# If your post title is longer or more complicated
+# than can be represented in the filename, uncomment the following line
+# and specify a custom title
+title:  "Bug Fix Contribution: Electric Vehicle Team"
+
+# Uncomment only one of the below categories
+categories: 
+- Bug Fix
+#- Contribution
```

Comment:
Also, can you update the category of your final contribution blog post so that it shows up in the contributions category on the site?
 ```suggestion
# - Bug Fix
- Contribution
```